static/app/components/events/interfaces/breadcrumbs/breadcrumb/type/icon.tsx

```diff
@@ -29,6 +32,12 @@ function Icon({type}: Props) {
       return <IconFix size="xs" />;
     case BreadcrumbType.INFO:
       return <IconInfo size="xs" />;
+    case BreadcrumbType.CLICK:
+      return <IconCursorArrow size="xs" />;
+    case BreadcrumbType.KEYDOWN:
+      return <IconKeyDown size="xs" />;
+    case BreadcrumbType.INPUT:
+      return <IconInput size="xs" />;
```

Comment:
This could affect the more general events/breadcrumb stuff. At least, it'll be hard to tell if regular breadcrumbs are using these new values.

I'd like to suggest if it's possible, to refactor so that `getFrameDetails` doesn't depend on `BreadcrumbType` anymore, and instead `getFrameDetails could import all the icons directly and return them. https://github.com/getsentry/team-replay/issues/122
What do you think?

Reply:
makes sense to me!